test/array_test.py

```diff
@@ -68,7 +72,9 @@ def testAssemble(self):
     ray.services.cleanup()
 
   def testMethods(self):
-    worker_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "test_worker.py")
+    for module in [ra.core, ra.random, ra.linalg, da.core, da.random, da.linalg]:
+      reload(module)
+    worker_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "../scripts/default_worker.py")
```

Comment:
It's the default for `start_ray_local`, but here we are calling `start_services_local` (which is not considered user facing) because we want to start two object stores.